lib/osf-components/addon/components/hero-banner/template.hbs

```diff
@@ -0,0 +1,51 @@
+<div local-class='hero'>
+    <header>
+        <h1 data-test-hero-heading>
+            {{t 'osf-components.hero-banner.heading'}}
+        </h1>
+        <p data-test-hero-subheading local-class='subheading'>
+            {{t 'osf-components.hero-banner.subheading'}}
+        </p>
+        {{#if this.shouldShowVersionB}}
+            <div local-class='flex-container'>
+                <div data-test-add-research-B local-class='addResearch versionB'>
+                    <h2 data-test-add-research-heading local-class='componentHeading versionB'>
+                        {{t 'osf-components.hero-banner.add_your_research.heading'}}
+                    </h2>
+                    <p data-test-add-research-subheading local-class='componentSubheading versionB'>
+                        {{t 'osf-components.hero-banner.add_your_research.subheading'}}
+                    </p>
+                    <GetStartedButton local-class='getStartedButton' />
+                </div>
+                <div data-test-discover-B local-class='discoverResearch versionB'>
+                    <h2 data-test-discover-heading local-class='componentHeading versionB'>
+                        {{t 'osf-components.search-bar.heading'}}
+                    </h2>
+                    <p data-test-discover-subheading local-class='componentSubheading versionB'>
+                        {{t 'osf-components.search-bar.subheading'}}
+                    </p>
+                    <SearchBar
+                        local-class='searchBar versionB'
+                        @onSearch={{action 'search'}}
+                    />
+                </div>
+            </div>
+        {{else}}
+            <div data-test-add-research-A local-class='addResearch'>
+                <GetStartedButton local-class='getStartedButton' />
+            </div>
+            <div data-test-discover-A local-class='discoverResearch'>
+                <h2 data-test-discover-heading local-class='componentHeading'>
+                    {{t 'osf-components.search-bar.heading'}}
+                </h2>
+                <p data-test-discover-subheading local-class='componentSubheading'>
+                    {{t 'osf-components.search-bar.subheading'}}
+                </p>
+                <SearchBar
+                    local-class='searchBar'
+                    @onSearch={{action 'search'}}
+                />
+            </div>
+        {{/if}}
```

Comment:
Would it be possible to dedupe this by always wrapping in a div and conditionally applying a `versionB` class to the container? Does that make the CSS too confusing? It would move all the versionB overrides to be together, which may be a good thing? Let's discuss.